docs/manage/security/saml/integrate-sumo-with-azure-ad.md

```diff
@@ -136,7 +140,33 @@ Do **not** close the **Setup Single Sign-On with SAML** window, you will return
 
 1. In the left navigation pane, click **Users and Groups** in the **Manage** section.
 1. Select **Add user/group**.
-1. Add the Users or Roles that should have access to login to Sumo Logic and then click **Assign**. 
+1. Add the Users or Groups that should have access to login to Sumo Logic and then click **Assign**. 
+
+### Configure Azure AD to send Group assignments to Sumo (Optional)
+
+In this step, you configure Azure AD to send group membership information in the SAML assertions it sends, so that Sumo Logic can assign roles to a user at each logon. This allows you to manage Sumo role assignments via Azure AD. If you don’t want to manage Sumo roles via Azure AD, skip these steps and proceed to [Test SAML Authentication](#test_saml_authentication) below.
```

Comment:
```suggestion
In this step, you configure Azure AD to send group membership information in the SAML assertions it sends, so that Sumo Logic can assign roles to a user at each logon. This allows you to manage Sumo role assignments via Azure AD. If you don’t want to manage Sumo roles via Azure AD, skip these steps and proceed to [Test SAML Authentication](#test_saml_authentication) below.
```
```suggestion
In this step, you configure Azure AD to send group membership information in the SAML assertions it sends, so that Sumo Logic can assign roles to a user at each logon. This allows you to manage Sumo role assignments via Azure AD. If you don’t want to manage Sumo roles via Azure AD, skip these steps and proceed to [Test SAML Authentication](#test-saml-authentication) below.
```